### ISSUE TITLE
Cannot run create_tasks.py for qa3 task

### ISSUE THREAD

**Issue by santiontanon** [booydar/babilong#7]
Hi, all other tasks I have tried work fine, but for qa3, it generates the 0k set fine, and then this happens when generating the 1k set:

```Traceback (most recent call last):
  File "/usr/local/home/santiontanon/my-research/babilong/data/./create_tasks.py", line 73, in <module>
    samples = [dataset_test[i] for i in inds]
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/home/santiontanon/my-research/babilong/data/./create_tasks.py", line 73, in <listcomp>
    samples = [dataset_test[i] for i in inds]
               ~~~~~~~~~~~~^^^
  File "/usr/local/home/santiontanon/my-research/babilong/data/../babilong/babilong_utils.py", line 181, in __getitem__
    sample = self.task_dataset[ind]
             ~~~~~~~~~~~~~~~~~^^^^^
  File "/usr/local/home/santiontanon/my-research/babilong/data/../babilong/babilong_utils.py", line 78, in __getitem__
    references = slc[slc.phrase_num.isin(slc.reference_num.values[-1])].text.values
                                         ~~~~~~~~~~~~~~~~~~~~~~~~^^^^
IndexError: index -1 is out of bounds for axis 0 with size 0
```

I just did git clone, installed prerequisites and ran `python create_tasks.py qa3_three-supporting-facts`

**Comment by booydar** [comment]
Hi, thanks for pointing this out! I made a fix in this commit:  [dab48b4](https://github.com/booydar/babilong/commit/dab48b4ec331857dcf1b494db020708986d79339)

**Comment by santiontanon** [comment]
Thanks! I just tested this and it works now! Thanks for the quick fix!
